.buildkite/pipeline.yml

```
@@ -49,3 +49,7 @@ steps:
       - *publish-android-artifacts-docker-container
     command: |
         .buildkite/publish-react-native-bridge-android-artifacts.sh
+
+  - label: "Run Danger CI"
```

It may be more beneficial to communicate the intent of this task, rather the implementation detail of Danger. WDYT? 
```suggestion
  - label: "Report mobile performance results"
```